src/test/java/org/katrin/JewelsCalculatorTest.java

```diff
@@ -0,0 +1,35 @@
+package org.katrin;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class JewelsCalculatorTest {
+    JewelsCalculator calc = new JewelsCalculator();
+
+    @Test
+    public void numJewelsInStonesOneLoopTest() {
+        String jewels = "aA";
+        String stones = "aAAbbbb";
+        int expected = 3;
+        int actual = calc.numJewelsInStonesOneLoop(jewels, stones);
+        Assert.assertEquals(expected, actual);
+    }
+
+    @Test
+    public void numJewelsInStonesTwoArraysLoopTest() {
+        String jewels = "aA";
+        String stones = "aAAbbbb";
+        int expected = 3;
+        int actual = calc.numJewelsInStonesTwoArrays(jewels, stones);
+        Assert.assertEquals(expected, actual);
+    }
+
+    @Test
+    public void numJewelsInStonesTwoLoopsTest() {
+        String jewels = "aA";
+        String stones = "aAAbbbb";
+        int expected = 3;
+        int actual = calc.numJewelsInStonesTwoLoops(jewels, stones);
+        Assert.assertEquals(expected, actual);
+    }
```

Comment:
this is a good approach for testing positives, negatives, and some edge cases. for example calculateJewels_JewelsIsEmpty_BadRequestIsThrown() {... }. Keep calling the Given-When-Then pattern. It would be clear what is being tested and what the expected result is